### DISCUSSION TITLE
Welcome to Strut Discussions!

### DISCUSSION THREAD

**Discussion by tantaman** [tantaman/Strut#395] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by josealbertogsi** [comment]
Hello!
I created a class for the text block, but I can't change the color via CSS.
How do I use CSS on Strut.io??
